index.html

```diff
@@ -825,6 +825,28 @@ <h2>
           for a web page using CSS.
         </p>
       </section>
+      <section class='informative'>
+        <h2>
+          Interaction with Web Content Accessibility Guidelines
+        </h2>
+        <p>
+            The Web Content Accessibility Guidelines 2.1 specification [[WCAG21]] includes a Success Criterion (<a href="https://www.w3.org/TR/WCAG21/#orientation">SC 1.3.4</a>) related to screen orientation.
+        </p>
+
+        <p>
+            ​The intent of this Success Criterion is to ensure that all content and functionality is available regardless of the display
```

Comment:
Suggest adding "essential" into this sentence: "...intent of this Success Criterion is to ensure that all essential content and functionality..."

Reply:
Hi @LJWatson ,

Essential is an undefined term here yet critical to the WCAG 2.1 spec, and I would be concerned about who determines what is essential. WCAG 2.1 has defined this term w.r.t. our spec, should this draft care to reference it: https://www.w3.org/TR/WCAG21/#dfn-essential

Again, it is recognized that this is non-normative text in this Specification, so defer to Editors judgement.